code/modules/games/cards.dm

```diff
@@ -61,36 +97,184 @@
 		build_deck()
 	deck_total = length(cards)
 
+/// Stub, override this to define how a base deck should be filled and built.
 /obj/item/deck/proc/build_deck()
 	return
 
+/obj/item/deck/proc/on_ranged_attack(mob/source, mob/attacker)
+	SIGNAL_HANDLER  // COMSIG_ATOM_RANGED_ATTACKED
+	INVOKE_ASYNC(src, TYPE_PROC_REF(/atom, attack_hand), attacker)
```

Comment:
Why is this invoking a proc only meant for adjacent turf interactions in a ranged attack?

Reply:
I want to make sure that, if you got here from this signal, any interactions will be identical to that of clicking it with a bare hand when adjacent.